extensions/media2/src/main/java/com/google/android/exoplayer2/ext/media2/PlayerWrapper.java

```diff
@@ -233,21 +233,13 @@ public boolean movePlaylistItem(
   public boolean skipToPreviousPlaylistItem() {
     Timeline timeline = player.getCurrentTimeline();
     Assertions.checkState(!timeline.isEmpty());
```

Comment:
Is there any value in this assertion? It doesn't look like other components (e.g., `PlayerControlView`) bother. If you can't see any value either, I'd suggest removing them (ditto for line `skipToNextPlaylistItem` below).

Reply:
I don't think they have any value either. Removed.

Reply:
Thanks. We will get this merged in.